shared/index.native.js

```diff
@@ -89,6 +89,13 @@ class Keybase extends Component {
 }
 
 function load () {
+  // Native String.startswith() sometimes incorrectly returns false on Android!
+  // $FlowIssue redefining startsWith
+  String.prototype.startsWith = function (searchString, position) { // eslint-disable-line no-extend-native
```

Comment:
also is this correct? maybe we should just use https://github.com/mathiasbynens/String.prototype.startsWith